localstack/plugin/discovery.py

```diff
@@ -0,0 +1,88 @@
+import importlib
+import inspect
+import logging
+import sys
+from types import ModuleType
+from typing import Iterable, List
+
+from .core import PluginFinder, PluginSpec, PluginSpecResolver
+
+LOG = logging.getLogger(__name__)
+
+
+class ModuleScanningPluginFinder(PluginFinder):
+    """
+    A PluginCollector that scans the members of given modules for available PluginSpecs. Each member is evaluated
+    with a PluginSpecResolver, and all successful calls resulting in a PluginSpec are collected and returned.
+    """
+
+    def __init__(self, modules: Iterable[ModuleType], resolver: PluginSpecResolver = None) -> None:
+        super().__init__()
+        self.modules = modules
+        self.resolver = resolver or PluginSpecResolver()
+
+    def find_plugins(self) -> List[PluginSpec]:
+        plugins = list()
+
+        for module in self.modules:
+            LOG.debug("scanning module %s", module.__name__)
+            members = inspect.getmembers(module)
+
+            for member in members:
+                if type(member) is tuple:
+                    try:
+                        spec = self.resolver.resolve(member[1])
+                        plugins.append(spec)
+                        LOG.info("found plugin spec in %s:%s %s", module.__name__, member[0], spec)
```

Comment:
since this is a call that will happen at build time i thought info would be fine. but `find_packages` also doesn't output everything it finds, so i suppose debug is the better option.